src/clients/SpokePoolClient.ts

```diff
@@ -331,7 +344,29 @@ export class SpokePoolClient extends BaseAbstractClient {
 
       return updatedDeposit;
     }
-    assert(false); // v3 is coming.
+
+    const v3SpeedUps = (this.speedUps[depositor]?.[depositId]?.filter(() => isV3SpeedUp) ?? []) as v3SpeedUp[];
```

Comment:
Do you think its worth storing v3 and v2 speedups separately in memory? Just a thought, im not opinionated

Reply:
The v2 | v3 union setup seems to work OK with speedups so far, so unless that proves to be flawed or difficult then I think we're OK to leave them consolidated.

Reply:
ok, i don' t think its a big decision